nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -796,6 +800,12 @@ enum op_band {
 	BAND_24G
 };
 
+enum rpu_feature_flags {
+	RPU_FEAT_TWT_EXTEND_SP_EDCA = BIT_POS(0),
+	RPU_FEAT_DISABLE_DFS_CHANNELS = BIT_POS(1),
+};
+
+/* TODO: For UMAC compatibility, remove once UMAC moves to latest enum*/
```

Comment:
Why are we doing it this way ? Shouldn't UMAC be making these changes first and then corresponding changes being made in the driver?

Reply:
@krish2718 : I have discussed this with @ajayparida and @srinivas1201 (@rajb9 FYI). I think this needs a overall cleanup in the UMAC header files. I have requested @ajayparida to file a enhancement Jira on the UMAC team (in which they can introduce a `BIT` macro and convert all relevant defines into `BIT` based enums). 

For this PR lets go ahead using the way it is being done presently in the UMAC header files (i.e. to have a #define instead of a BIT based enum).

Reply:
> Why are we doing it this way ? Shouldn't UMAC be making these changes first and then corresponding changes being made in the driver?

No, our process is reverse, driver does the changes and we backport them to UMAC.

Reply:
The enhanement work being tracked under task [SHEL-2391](https://nordicsemi.atlassian.net/browse/SHEL-2391).